### PR TITLE
Bump version to 0.24.1

### PR DESCRIPTION
**Pull request by pipo02mix**
Upgrade nginx controller version to `0.24.1`. It comes with a new nginx version and solved an issue with the default certificates.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+
+## [0.6.0]
+
+### Added
+
+- Nginx kubectl plugin has added a commend to `lint` your ingress resources. That way you can observe deprecated annotations or properties. To install the command go [here](https://github.com/kubernetes/ingress-nginx/blob/29f7d9a77ade24a7366ef4a6f258b8aeef50678c/docs/kubectl-plugin.md).
```
Comment (puja108) [inline CHANGELOG.md]:
```suggestion
- Nginx kubectl plugin has a new command to `lint` your ingress resources. That way you can observe deprecated annotations or properties. To install the plugin go [here](https://github.com/kubernetes/ingress-nginx/blob/29f7d9a77ade24a7366ef4a6f258b8aeef50678c/docs/kubectl-plugin.md).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+### Changed
+
```
Comment (puja108) [inline CHANGELOG.md]: the two changes should be below a general change that the version of the IC has been updated to 0.24.1, those two are details of the changelog of that, not changes we made

**Review by puja108 — COMMENTED** [review]
minor typo and needs addition of actual upstream version needed

File: CHANGELOG.md (hunk 2 above)
Comment (puja108) [reply CHANGELOG.md]: also don't forget to link to upstream changelog/release notes for that

Comment (pipo02mix) [reply CHANGELOG.md]: But what do you think about mention it here? or just point to the upstream changelog?

Comment (puja108) [reply CHANGELOG.md]: good but as sub points of the upstream change, like "notable changes"

**Comment by pipo02mix** [comment]
Changelog reformatted

**Review by puja108 — APPROVED** [review]

**Pull request merged**